### PR TITLE
Readme config section

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL

File: README.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ We are working on expanding the types of frameworks and services we can do autom
 
 We also have an API that lets you add custom spans to the traces automatically. This can be useful if your application does sizable synchronous or asynchronous work that is not using an existing IO or RPC module.
 
+### (Optional) Trace Configuration
+
+The trace agent can be configured by passing a configurations object to the agent `start` method. This configuration option accepts all values in the [default configuration](config.js).
+
+One configuration option of note is `enhancedDatabaseReporting`. Setting this option to `true` will cause database operations for redis and MongoDB to record queries and results as labels on reported trace spans.
```
Comment (ofrobots) [inline README.md]: It might be worthwhile pointing out that it would be summary results instead of full query results that will be stored.

**Comment by ofrobots** [comment]
lgtm w/ comment.

File: README.md (hunk 1 above)
Comment (matthewloring) [inline README.md]: Fixed

**Pull request merged**